cmd/snap/cmd_snap_op.go

```diff
@@ -146,6 +147,7 @@ func (x *cmdInstall) Execute([]string) error {
 
 type cmdRefresh struct {
 	Channel    string `long:"channel" description:"Refresh to the latest on this channel, and track this channel henceforth"`
+	DevMode    bool   `long:"devmode" description:"Refresh the snap with non-enforcing security"`
```

Comment:
Let's please remove this from refresh for now. The use case we understand is local development, and the store should have properly confined snaps only.